src/m365/spfx/commands/spfx-doctor.ts

```diff
@@ -1,6 +1,9 @@
 import * as chalk from 'chalk';
 import * as child_process from 'child_process';
+import { resolve } from 'dns';
```

Comment:
Why do we need these imports on line 3, 4 and 6?

Reply:
I am not sure why those got added. I did not do it directly, but suspect it happened as I was trying to sync my fork to pnp. I think those were taken out at some point?  I will remove them

Reply:
Let's indeed ensure that your PR doesn't contain anything that we shouldn't include.